index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet"  href="styles.css">
+
+</head>
+<body>
+    <div class="wrapper">
+            <!-- <div class="nav-bar">
+                <P>11:22</P>
+                <div class="utility-icons">
+                    <img src="/assets/cellular-connection.png" alt="tower reception">
+                    <img src="/assets/wifi.png" alt="wifi strengh"> 
+                    <img src="/assets/battery.png" alt="battery level">
+                </div>
+            </div> -->
+            
+            <header>
+                <div class="logo">
```

Comment:
- [ ] you did a great job so far on placing a logo in the design ㊗️  but you should wrap your logo inside an `a` tag to make your project more dynamic instead of using a `div` tag which also makes your code more semantic which shows professionalism in coding.